src/Service.Tests/OpenApiDocumentor/ParameterValidationTests.cs

```diff
@@ -0,0 +1,176 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Azure.DataApiBuilder.Config.ObjectModel;
+using Microsoft.IdentityModel.Tokens;
+using Microsoft.OpenApi.Models;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+namespace Azure.DataApiBuilder.Service.Tests.OpenApiIntegration;
+
+/// <summary>
+/// Integration tests validating correct path and query parameters are created
+/// for different entities in DAB's REST API endpoint.
+/// </summary>
+[TestCategory(TestCategory.MSSQL)]
+[TestClass]
+public class ParameterValidationTests
+{
+    private const string CUSTOM_CONFIG = "parameter-config.MsSql.json";
+    private const string MSSQL_ENVIRONMENT = TestCategory.MSSQL;
+
+    /// <summary>
+    /// Validates the path parameters are correctly generated for GET methods.
```

Comment:
What would constitute 'incorrectly' generated path parameters?

Reply:
ahaha, right. I'll update the description.

Reply:
But by incorrectly it could be like no pathParameter is there